Python/cp/LetterCount.py

```diff
@@ -0,0 +1,36 @@
+"""
+The program uses dictionary in python
+to count the number of occurence a letter
+makes in a word
+"""
+
+
+def LetterCount(word):
+    dict = {}
+
+    for c in word:
+        co = dict.get(c, 0)
+        if co == 0:
+            dict[c] = 1
+        else:
+            dict[c] = co + 1
+
+    return(dict)
+
+
+word = input("enter a word ").lower()
+
+LetterCount = LetterCount(word)
+
+print(LetterCount)
```

Comment:
Enclose in a `if __name__ == '__main__'` block

Reply:
Done @HarshCasper